core/src/main/java/org/apache/accumulo/core/spi/scan/ConfigurableScanServerSelector.java

```diff
@@ -124,7 +125,7 @@
  *       "maxBusyTimeout":"20m",
  *       "busyTimeoutMultiplier":8,
  *       "group":"lowcost",
- *       "enableTabletServerFallback":false,
+ *       "timeToWaitForScanServers": "120ms,
```

Comment:
120ms seems low to me for an example, but this is subjective feel free to ignore this comment.

```suggestion
 *       "timeToWaitForScanServers": "120s,
```